benchexec/tools/ltsmin.py

```diff
@@ -0,0 +1,43 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2025 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.result as result
+from benchexec.tools.template import BaseTool2
+
+
+class Tool(BaseTool2):
+    """
+    Wrapper for LTSmin.
+    """
+
+    REQUIRED_PATHS = ["bin/", "include/", "share/"]
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("ltsmin-reduce", subdir="bin")
```

Comment:
Is `ltsmin-reduce` a meaningful default? I.e., if users specify only the tool-info module, will they expect `ltsmin-reduce` as default?